db/schema.rb

```diff
@@ -82,11 +82,14 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer "role", default: 0
+    t.bigint "merchant_id"
+    t.index ["merchant_id"], name: "index_users_on_merchant_id"
```

Comment:
Having role and merchant affiliation will help employees interact with specific items!